tests/test_views.py

```diff
@@ -3208,6 +3208,43 @@ def setUpClass(cls):
         cls._original_enable_email_subscriptions = settings.ENABLE_EMAIL_SUBSCRIPTIONS
         settings.ENABLE_EMAIL_SUBSCRIPTIONS = True
 
+    def test_user_unsubscribe_and_subscribe_help_mailing_list(self):
+        user = AuthUserFactory()
+        url = api_url_for('user_choose_mailing_lists')
+        payload = {settings.OSF_GENERAL_LIST: False}
+        res = self.app.post_json(url, payload, auth=user.auth)
+        user.reload()
+
+        assert_false(user.osf_mailing_lists[settings.OSF_GENERAL_LIST])
+        assert_equal(
+            user.osf_mailing_lists[settings.OSF_GENERAL_LIST],
+            payload[settings.OSF_GENERAL_LIST]
+        )
+
+        payload = {settings.OSF_GENERAL_LIST: True}
+        res = self.app.post_json(url, payload, auth=user.auth)
+        user.reload()
+
+        assert_true(user.osf_mailing_lists[settings.OSF_GENERAL_LIST])
+        assert_equal(
```

Comment:
This was done to follow convention set by the mailchimp subscribe test, but it can be removed.